README.md

```diff
@@ -6,37 +6,83 @@
+* [Build and run DICOM Viewer for development](#Build-and-run-for-development)
+* [Contributing to this app](#Contributing-to-this-app)
+* [Copyright](#Copyright)
 
-- **Supported architectures:**\
-  `amd64`
 
-## Installation
+## Functionalities of the DICOM Viewer Web Extension
 
-### 1. Install dependencies
+The current release allows to preview .dcm files within oCIS and display their corresponding metadata in a sidebar on request. It offers image manipulation operations such as zoom in and out, rotation, flipping, colour inversion and reset on the image preview. The app UI is implemented in a responsive manner and adapts the size of the image preview and the way how metadata is displayed to the screen size of the device.
 
-```
+<img src="https://github.com/owncloud/awesome-ocis/blob/main/webApps/owncloud/web-app-dicom-viewer/screenshots/1.png" alt="app functionalities" style="width:48%; height:auto;"> &nbsp; &nbsp; <img src="https://github.com/owncloud/awesome-ocis/blob/main/webApps/owncloud/web-app-dicom-viewer/screenshots/4.png" alt="display of metadata" style="width:48%; height:auto;">
+
+_The extension allows to preview a DICOM image and it's most important metadata. On request, all corresponding metadata of the file are displayed in the sidebar._
+
+<img src="https://github.com/owncloud/awesome-ocis/blob/main/webApps/owncloud/web-app-dicom-viewer/screenshots/2.png" alt="app functionalities" style="width:48%; height:auto;"> &nbsp; &nbsp; <img src="https://github.com/owncloud/awesome-ocis/blob/main/webApps/owncloud/web-app-dicom-viewer/screenshots/3.png" alt="display of metadata" style="width:48%; height:auto;">
+
+_The extension allows to zoom, rotate and flip the preview of the image. Inverting the colors of the preview is also supported._
+
+
+## Adding DICOM Viewer to your oCIS installation
+As oCIS administrator, you can add custom web applications for your users. By adding the DICOM Viewer to the oCIS WebUI, you enable your users to take advantage of the [functionalities of this web extension](#Functionalities-of-the-DICOM-Viewer-Web-Extension).
+
+Have a look at the ownCloud Infinite Scale Deployment documentation to learn how to [extend the WebUI with apps](https://doc.owncloud.com/ocis/next/deployment/webui/webui-customisation.html#extend-web-ui-with-apps). You will find instructions how to [load custom applications](https://doc.owncloud.com/ocis/next/deployment/webui/webui-customisation.html#loading-applications) into your installation and get a better understanding of the web extension [application structure](https://doc.owncloud.com/ocis/next/deployment/webui/webui-customisation.html#application-structure) and [application configuration](https://doc.owncloud.com/ocis/next/deployment/webui/webui-customisation.html#application-configuration).
+
+---
+
+DICOM Viewer's `manifest.json` (referenced in `application structure`) can be found [here](public/manifest.json).
+
+---
+
```

Comment:
TODO:
- find out if `apps.yaml` file is needed (seems to be optional)
- if so, provide corresponding file and link to it?
- see draft PR https://github.com/owncloud/web-app-dicom-viewer/pull/67

Reply:
@mmattel from our understanding yaml is optional. We have been running our web extensions on a server following the installation instructions given here respectively in the README of the other extensions we've developed without any yaml file